var/spack/repos/builtin/packages/py-opppy/package.py

```diff
@@ -11,20 +11,21 @@ class PyOpppy(PythonPackage):
     analysis library designed to extract, store, and plot information from
     output and dump files generated by scientific software packages."""
 
-    homepage = "https://github.com/lanl/opppy"
-    url = "https://github.com/lanl/OPPPY/archive/opppy-0_1_2.tar.gz"
-    git = "https://github.com/lanl/opppy.git"
+    homepage = "https://github.com/lanl/OPPPY"
+    url = " https://github.com/lanl/OPPPY/archive/refs/tags/opppy-0_1_1.tar.gz"
```

Comment:
```suggestion
    url = "https://github.com/lanl/OPPPY/archive/refs/tags/opppy-0_1_1.tar.gz"
```
I think you also need to override the URL for 0_1_1 or add a `url_for_version` function. You also need to update the checksums for older versions (can be done using `spack checksum`).

Reply:
> I think you also need to override the URL for 0_1_1 or add a `url_for_version` function. You also need to update the checksums for older versions (can be done using `spack checksum`).

Yes!  The difference is the tarball for `0_1_1` uses upper case `OPPPY` while the rest of the releases are lower case.

Reply:
Nice catch. I think I cleaned this up following version url examples in the documentation.

Reply:
All versions appear to work now using spack checksum (I didn't know about this handy command).